dot/state/storage_notify_test.go

```diff
@@ -173,16 +171,6 @@ func Test_Example(t *testing.T) {
 	bValue := []byte("b-value")
 
 	// Open the DB.
-	dir, err := ioutil.TempDir("", "badger-test")
-	if err != nil {
-		log.Fatal(err)
-	}
-	defer func() {
-		if err = os.RemoveAll(dir); err != nil {
-			log.Fatal(err)
-		}
-	}()
-
```

Comment:
@EclesioMeloJunior Is this removal harmless?

Reply:
Yes since `NewInMemoryDB` uses `t.TempDir` which removes the directory when the test finishes.